util/maintenance.php

```diff
@@ -49,5 +49,5 @@
 }
 
 echo "Usage:\n\n";
-echo "\tphp {$argv[0]} [1] [Maintenance reason]\tSet the system in maintenance mode\n";
+echo "\tphp {$argv[0]} [1] [Maintenance reason|redirection url]\tSet the system in maintenance mode\n";
```

Comment:
Maybe below the examples as a test paragraph? One blank line and then the explanation text.